TSG_list.txt

```diff
@@ -311,4 +311,4 @@ ZMYM3
 ZNF278
 ZNF331
 ZNRF3
-ZRSR2
```

Comment:
Not sure what the change was here, but looks fine.

Reply:
I think the last line was empty and it got removed.